Documentation/RenderingDocs/Index.rst

```diff
@@ -10,97 +10,116 @@
 .. _render-documenation-with-docker:
 .. _render-documentation-with-docker:
 
+=========================================
+Render documentation with the TYPO3 theme
+=========================================
 
-===========================
-How to render documentation
-===========================
+..  contents::
 
-This page explains how to render a manual locally on your machine
-with Docker in order to test the rendering.
+Rendering the :file:`Documentation` folder locally with Docker
+==============================================================
 
-Run these commands in a terminal in the parent directory of the directory
-:file:`Documentation`. You should use a bash compatible shell, if possible.
+You can render the documentation of an official TYPO3 manual or a third party
+manual with the following steps:
 
-If you run into a problem while rendering, check :ref:`rendering-docs-troubleshooting`,
-`report an issue <https://github.com/t3docs/docker-render-documentation/issues/new>`__
-or :ref:`ask for help <get-help-on-writing-docs>`.
+#.  Clone the repository containing the documentation.
 
+#.  Navigate to the directory, which contains the :file:`composer.json`
```

Comment:
Why composer.json and not just project root or similar?

Reply:
Just to give a reference what we are talking about. If for example you are in the TYPO3 monorepository you would have to be in the sysextensions folder - just where the composer.josn ist

Reply:
adjusted the text